Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -648,6 +655,8 @@ def _uniqueFirstLayers(layerList):
 tracksValidationTruth = cms.Task(
     tpClusterProducer,
     tpClusterProducerPreSplitting,
+    trackAssociatorByChi2,
+    MTVTrackAssociationByChi2,
```

Comment:
do we need these 2 modules in production ?

Reply:
No, with the current configuration (are part of the setup of MTV to be able to easily switch to associator by chi2, but if you think they create mess I remove it... For my understanding, they are not running on anything).